sphinx/source/docs/user_guide/charts.rst

```diff
@@ -241,6 +241,14 @@ Bar
 
   * ``True``: bars are draw as a stack to show the relationship of parts to a whole.
   * ``False``: bars are groupped on the same chart figure. Defaults to ``False``.
+* ``y_range`` (Range, optional): supply a Range object for the y_range 
```

Comment:
Suggest:
```
* ``y_range`` (Range, optional):  An explicit range for the y-dimension
```
In general I try to construct all parameter descriptions to be readable with an implicit "IS", e.g.:
```
* y_range IS An explicit range for the y-dimension
```